rmc/resources/basics.py

```diff
@@ -1,405 +1,109 @@
-    },
-)
-"""
-Table containing all transcripts with evidence of regional missense constraint.
-
-Contains transcripts with one or additional breaks plus simultaneous breaks results.
-"""
-
-context_with_oe = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(path=f"{CONSTRAINT_PREFIX}/{version}/context_with_oe.ht")
-        for version in GNOMAD_VERSIONS
-    },
-)
-"""
-Table containing missense variants in canonical transcripts annotated with missense OE.
-
-Each row is annotated with RMC transcript subsection missense OE if it exists, otherwise
-the transcript level missense OE.
-"""
-
-context_with_oe_dedup = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(
-            path=f"{CONSTRAINT_PREFIX}/{version}/context_with_oe_dedup.ht"
-        )
-        for version in GNOMAD_VERSIONS
-    },
-)
-"""
-Deduplicated version of `context_with_oe`.
-
-Some locus/alleles combinations are present more than once in `context_with_oe` if they have
-a most severe consequence of 'missense_variant' in more than one canonical transcript.
-
-This Table contains only one row per each unique locus/alleles combination.
-"""
-
-rmc_browser = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(
-            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/rmc_browser.ht"
-        )
-        for version in GNOMAD_VERSIONS
-    },
-)
+CONSTRAINT_PREFIX = f"{RMC_PREFIX}/constraint"
 """
-Table containing all transcripts with evidence of regional missense constraint.
-
-Contains same information as `rmc_results` but has different formatting for gnomAD browser.
+Path to bucket containing RMC output Tables.
 """
 
-amino_acids_oe = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(path=f"{MPC_PREFIX}/{version}/amino_acid_oe.ht")
-        for version in GNOMAD_VERSIONS
-    },
-)
+MPC_PREFIX = f"{RMC_PREFIX}/MPC"
 """
-Table containing all possible amino acid substitutions and their missense OE ratio.
-
-Input to missense badness calculations.
+Path to bucket containing resources related to building MPC score.
 """
 
-misbad = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(path=f"{MPC_PREFIX}/{version}/missense_badness.ht")
-        for version in GNOMAD_VERSIONS
-    },
-)
-"""
-Table containing all possible amino acid substitutions and their missense badness scores.
-"""
 
-joint_clinvar_gnomad = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(path=f"{MPC_PREFIX}/{version}/joint_clinvar_gnomad.ht")
-        for version in GNOMAD_VERSIONS
-    },
-)
+######################################################################
+## Amino acid resources
+######################################################################
```

Comment:
I think it would make sense to make a variable
```
AMINO_ACIDS_PREFIX = f"{RESOURCE_PREFIX}/amino_acids"
```
or something similar and then use that for each of these resources here.